zhmccli/_cmd_partition.py

```diff
@@ -368,15 +399,21 @@ def partition_dump(cmd_ctx, cpc, partition, **options):
 @optgroup.option('--access-extended-counter-set', type=bool, required=False,
                  help='Indicates if extended counter set authorization control '
                  'is requested. Default: False')
-@optgroup.option('--access-coprocessor-group-set', type=bool, required=False,
```

Comment:
This property on the API never had a function. I would propose to remove it from here.

Reply:
agreed